README.md

```diff
@@ -6,13 +6,101 @@
 
 This extension aims to support the creation of **Liquid templates** with the [Silverfin Templating Language](https://developer.silverfin.com/docs) and the development of [Liquid testing YAML](https://developer.silverfin.com/docs/liquid-testing) files.
 
+## Setup & Basic Usage
+
+### 3rd Party Prerequisites
```

Comment:
I'd rename this to "Third party extensions" and move it to the bottom of our documentation (since it's just ter info and to give attribution)

Reply:
Agreed and moved to bottom of doc